src/routes/safe/components/Balances/Tokens/screens/AssetsList/index.tsx

```diff
@@ -126,12 +127,13 @@ const AssetsList = (props) => {
         </Row>
         <Hairline />
       </Block>
-      {!nftAssetsList.length && (
-        <Block className={classes.progressContainer} justify="center">
-          <Paragraph>No collectibles available</Paragraph>
-        </Block>
-      )}
-      {nftAssetsList.length > 0 && (
+      {!nftAssetsList.length ||
```

Comment:
Syntax is easier to follow if is written like this:
```
{(!nftAssetsList || !nftAssetsList.length) && (
    <Block className={classes.progressContainer} justify="center">
        {!nftAssetsList ? <CircularProgress /> : <Paragraph>No collectibles available</Paragraph>}
     </Block>
)}
```

Reply:
`{(!nftAssetsList?.length)`

Reply:
It will also work :+1: